find.go

```diff
@@ -129,21 +129,21 @@ func (f *finder) structType(name *ast.Ident, t *ast.StructType) {
 	}
 }
 
-const _required = "// required"
+const _required = "required"
 
 func isRequiredComment(c *ast.Comment) bool {
-	if c.Text == _required {
+	text := strings.TrimPrefix(c.Text, "//")
```

Comment:
With this change, we're no longer verifying that the string starts with `//`

Should we use `CutPrefix` and verify that there was a `//` prefix? Since this is a comment node, I'm not sure if that's required